packages/oid4vci-demo-frontend/public/locales/development/en/translation.json

```diff
@@ -33,23 +33,24 @@
   "qrcode_right_pane_top_paragraph": "A hassle-free way to share your personal\ninformation using your digital wallet.",
   "qrcode_right_pane_top_title": "Use your Digital Identity",
   "sharing_data_right_pane_paragraph": "Almost there!\n Please fill out any missing data.\r\nTo claim your {{credentialName}}\ncredential(s). The content will not be stored! It is used to personalize the credentials",
-  "sharing_data_fill_out_right_pane_paragraph": "To get started, please provide us with your details",
+  "sharing_data_fill_out_right_pane_paragraph": "To get started, please provide us with your personal details",
```

Comment:
This will break the knb demo. Please revert and use resource "title_personal_details" (which was already created for account flow design)
If there is no resource with the text you need, create a new one.

The same for other resources, you can't change those unless it has to change for all ecosystems.